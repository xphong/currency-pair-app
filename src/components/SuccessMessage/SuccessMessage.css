.success {
  color: green;
  margin: 12px 0;
}
