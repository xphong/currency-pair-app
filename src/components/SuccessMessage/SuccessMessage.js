import React from 'react';

import styles from './SuccessMessage.css';

const SuccessMessage = props => {
  return (
    <div className={styles.success}>
      {props.children}
    </div>
  );
};

export default SuccessMessage;
