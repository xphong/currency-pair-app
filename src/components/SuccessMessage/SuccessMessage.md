### Default Style

```js
<SuccessMessage>Success!</SuccessMessage>
```
