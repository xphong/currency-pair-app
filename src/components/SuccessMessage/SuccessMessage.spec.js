import React from 'react';

import SuccessMessage from './SuccessMessage';

describe('SuccessMessage Component', () => {
  it('should render SuccessMessage', () => {
    const wrapper = shallow(<SuccessMessage>success</SuccessMessage>);

    expect(wrapper).toMatchSnapshot();
  });
});
