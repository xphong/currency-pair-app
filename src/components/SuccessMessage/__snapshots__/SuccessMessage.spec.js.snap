// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SuccessMessage Component should render SuccessMessage 1`] = `
<div
  className="success"
>
  success
</div>
`;
